doc/news/_preparation_next_release.md

```diff
@@ -283,6 +283,7 @@ This section keeps you up-to-date with the multi-language support provided by El
 - <<TODO>>
 - Fix grammar for `elektra-granularity.md` _(@dtdirect)_
 - Rephrase sections in doc/dev/error-\* _(@dtdirect)_
+- Extend check_doc.sh to work for contrib and tutorials _(@Joni1993)_
```

Comment:
I believe this should work:

```suggestion
- Extend check\_doc.sh to work for contrib and tutorials _(@Joni1993)_
```

Alternatively, try this:

```suggestion
- Extend `check_doc.sh` to work for contrib and tutorials _(@Joni1993)_
```

Reply:
> - Extend check\_doc.sh to work for contrib and tutorials _(@Joni1993)_

Had this before - restyled suggested this patch: https://restyled.io/gh/ElektraInitiative/repos/libelektra/jobs/2485877/patch

Reply:
Discussion moved to: #4687